extensions/typescript-language-features/src/features/fileConfigurationManager.ts

```diff
@@ -165,6 +165,7 @@ export default class FileConfigurationManager extends Disposable {
 			insertSpaceAfterTypeAssertion: config.get<boolean>('insertSpaceAfterTypeAssertion'),
 			placeOpenBraceOnNewLineForFunctions: config.get<boolean>('placeOpenBraceOnNewLineForFunctions'),
 			placeOpenBraceOnNewLineForControlBlocks: config.get<boolean>('placeOpenBraceOnNewLineForControlBlocks'),
+			semicolons: config.get<Proto.SemicolonPreference>('semicolons'),
```

Comment:
So that we can merge this before VS Code includes TS 3.7, just use a normal string for now (or inline the string literal types)

Reply:
Done. I had to modify the type of the object to get away with adding the additional property. Left a note that it can be removed after 3.7 is adopted.